Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN addgroup -S wegajetty \
 
 ARG VERSION
 ARG MAX_MEMORY
+ARG UPDATE_XARS
```

Comment:
I think this is unnecessary. Why would you want to have old versions (as default)? IMHO this ARG can be removed completely.

Reply:
If you want to add some custom XARs (see below) it might be nice to not autoupdate them as the script at the moment flushes all contents of the auto deploy folder.